src/modules/geolocation/geolocation-directive.js

```diff
@@ -146,8 +146,8 @@ angular.module('anol.geolocation')
                         geolocation.setTracking(false);
                         const position = geolocation.getPosition();
                         const accuracyGeometry = geolocation.getAccuracyGeometry();
-                        const constrainedPosition = view.adjustCenter(position);
-                        if (position[0] !== constrainedPosition[0] || position[1] !== constrainedPosition[1]) {
+                        const extent = view.getProperties().extent;
```

Comment:
The `getProperties` method in openlayers is usually used for custom properties, for which openlayers does not provide an API. In order to get the extent of the current view, it is recommended to use the [calculateExtent](https://openlayers.org/en/latest/apidoc/module-ol_View-View.html#calculateExtent) method of the View class.